Source/Core/Core/PowerPC/JitCommon/JitCache.cpp

```diff
@@ -183,7 +182,26 @@ JitBlock* JitBaseBlockCache::GetBlockFromStartAddress(u32 addr, u32 msr)
 const u8* JitBaseBlockCache::Dispatch()
 {
   const auto& ppc_state = m_jit.m_ppc_state;
-  JitBlock* block = m_fast_block_map_ptr[FastLookupIndexForAddress(ppc_state.pc)];
+  if (m_entry_points_ptr)
+  {
+    if (m_entry_points_ptr[FastLookupIndexForAddress(ppc_state.pc, ppc_state.msr.Hex)])
+    {
+      return m_entry_points_ptr[FastLookupIndexForAddress(ppc_state.pc, ppc_state.msr.Hex)];
```

Comment:
I'd suggest putting `m_entry_points_ptr[FastLookupIndexForAddress(ppc_state.pc, ppc_state.msr.Hex)]` in a variable so it doesn't have to be evaluated twice. (Not sure if the compiler is smart enough to figure out that `FastLookupIndexForAddress` has no side effects.)